entity/src/main/java/org/grnet/cat/entities/Assessment.java

```diff
@@ -50,4 +52,8 @@ public class Assessment {
 
     @Column(name = "updated_by")
     private String updatedBy;
+
+    @OneToMany(mappedBy = "assessment")
```

Comment:
It would be best if you turned one-to-many relation into the following:
 @OneToMany(
            mappedBy = "assessment",
            cascade = CascadeType.ALL,
            orphanRemoval = true
    )